app/src/main/res/values/theme.xml

```diff
@@ -54,4 +54,9 @@
         <item name="colorAccent">@color/accent</item>
     </style>
 
+    <style name="Theme.GitHub.Alert.Dialog.Style" parent="Theme.AppCompat.Light.Dialog.Alert">
+        <item name="colorAccent">@color/accent</item>
```

Comment:
why are you setting colorAccent to @color/accent again? its already set in the main theme